src/Microsoft.Identity.Web.DownstreamApi/DownstreamApi.cs

```diff
@@ -336,7 +356,19 @@ internal partial class DownstreamApi : IDownstreamApi
             // which is done by not specifying any scopes
             if (effectiveOptions.Scopes != null && effectiveOptions.Scopes.Any())
             {
-                string authorizationHeader = appToken ?
+                string? authorizationHeader;
+                if (_authorizationHeaderProviderExtension != null)
+                {
+                    authorizationHeader = await _authorizationHeaderProviderExtension.CreateAuthorizationHeaderAsync(
+                       new RequestContext(),

```

Comment:
How is RequestContext used?

Reply:
RequestContext is not needed today. It was added for future use. For example, if serviceName needs to be passed down from DownstreamApi to AuthorizationHeaderProvider implementation.

Reply:
RequestContext has been removed as agreed offline as there is no need for it today.